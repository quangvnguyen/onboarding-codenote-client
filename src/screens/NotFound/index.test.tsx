import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './index';

test('renders without crashing', () => {
  const wrapper = shallow(
    <NotFound />
  );
  expect(wrapper).toMatchSnapshot();
});
