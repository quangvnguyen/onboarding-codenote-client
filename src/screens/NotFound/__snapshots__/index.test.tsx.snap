// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="NotFound"
>
  <h3>
    Sorry, page not found!
  </h3>
</div>
`;
